modules/navier_stokes/include/base/NS.h

```diff
@@ -134,6 +134,12 @@ static const std::string c = "c";
 static const std::string speed = "speed";
 static const std::string sound_speed = "sound_speed";
 
+// Two phase mixture materials
+static const std::string latent_heat = "latent_heat";
+static const std::string T_liquidus = "T_liquidus";
+static const std::string T_solidus = "T_solidus";
+static const std::string alpha_exc = "alpha_exc";
```

Comment:
I think I would like "exchange" better. "exc" makes me think of execution

Reply:
changed